packages/components/src/select-control/stories/index.story.tsx

```diff
@@ -64,6 +65,7 @@ const SelectControlWithState: StoryFn< typeof SelectControl > = ( props ) => {
 export const Default = SelectControlWithState.bind( {} );
 Default.args = {
 	__nextHasNoMarginBottom: true,
+	label: 'Label',
```

Comment:
While we're here — I noticed that we don't add labels in our stories, which is bad for accessibility. Let's add them.